src/monorepo-npm-publisher/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+- Upgrade `@adeira/babel-preset-adeira` to 2.0.0.
+- Add reactRuntime option. You can set it to `automatic` to use the new [JSX transform](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html), or set it to `classic` to use the classical version. Note that `automatic` is the default.
```

Comment:
This is a breaking change, right?

Reply:
Yes, that is correct. The next release should be a major version. Should I mark it here with BC?

Reply:
Yes please, just to be sure we release it correctly.

Reply:
🚀